.travis.yml

```diff
@@ -1,21 +1,14 @@
+# ref: https://docs.travis-ci.com/user/languages/python
```

Comment:
Let's keep the previous version instead for now, until we switch to `CircleCI`.

The `script` line might need to be tweaked though (depending on how you write the test suite).